utils/nft-collection.ts

```diff
@@ -157,4 +166,13 @@ export class NFTCollection extends DeployHelpers {
       }
     )
   }
+}
+
+async function calcContractId(txId: string, groupIndex: number) {
```

Comment:
Maybe not part of this PR, but feels like this could be moved to SDK

Reply:
Sounds great 👍, will move it to SDK.